samples/helloworldasync/helloworld.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
```

Comment:
Same comment, do you need these conditionals?

Reply:
Also async main was added in C# 7.1 I believe. Do you even need this Property?

Reply:
Conditions taken from other github issue when faced the problem with compiler, but just checked without them and works well. Will change to ```latest``` instead of 7.2.
And yes, ```async``` was available since 7.1.